@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/ihadss/fn_Ihadsstoggle.sqf

```diff
@@ -0,0 +1,17 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fnc_Ihadsstoggle
+Description:
+Parameters:
+Returns:
+    Nothing
+    
+Examples:
+    [_heli] spawn fza_fnc_Ihadsstoggle;
```

Comment:
This code doesn't have any sleeps, it is callable. There are a few more functions like this so go through and if it doesn't have any long loops / sleeps / waitUntils then use `call` rather than `spawn`